extensions/desktop/common/desktop-exports

```diff
@@ -467,3 +467,6 @@ ln -sfn "$REALHOME/.config/ibus/bus" "$IBUS_CONFIG_PATH"
 
 # Set libgweather path
 export LIBGWEATHER_LOCATIONS_PATH="$SNAP_DESKTOP_RUNTIME/usr/lib/$ARCH/libgweather-4/Locations.bin"
+
+# Set libthai dict path
+export LIBTHAI_DICTDIR="$SNAP/gnome-platform/usr/share/libthai/"
```

Comment:
Hmm... shouldn't this need to be `$SNAP_DESKTOP_RUNTIME` instead, in case the runtime used is KDE?

Reply:
so `$SNAP/$SNAP_DESKTOP_RUNTIME/usr/share/libthai/` ?

Reply:
Judging from the line above, I think it's more likely just `$SNAP_DESKTOP_RUNTIME/usr/share/libthai/`.

Note that it's not as important for KDE runtime, as Qt seems to use ICU for Thai word breaking. However, KDE runtime _does_ include libthai, so it's possible that snap'ed application uses libthai from KDE runtime and run into the same problem.